test_pii_data.py

```diff
@@ -104,6 +104,10 @@ def test_has_ipv6(self):
         test_data = Pii('2001,0db8,0001,0000,0000,0ab9,C0A8,0102')
         self.assertFalse(test_data.has_ipv6())
 
+    def test_has_ipv6_anonymize(self):
```

Comment:
This test does not pass for 2 reasons.  

1) The regex forces the beginning of a string with the ^ character - if you remove that it will match correctly 

2) Your replacement is [ipv6] but your test case looks for [IPv6] so it's case sensitive